### ISSUE TITLE
Connection problems

### ISSUE THREAD

**Issue by alexeyzab** [urbit/urbit#1301]
I am having pretty much the exact issues outlined in https://github.com/urbit/urbit/issues/1234, except I am running urbit `0.7.4`.

I am able to `|hi ~zod` successfully, but `|hi ~ten` doesn't return a response at all. The same thing with `|hi ~dopzod`, I can't join `~dozpod/urbit-help`.

I've tried doing the `|nuke` trick to block and unblock the connection, it didn't work, I've tried resetting my public keys multiple times too but that didn't help. Could it be that my networking keys need some adjustments? I couldn't find a way to regenerate those in Bridge, wasn't sure if that functionality got removed or not.

I thought that my point's (`~fodbyr-nosmyr`) parent star was not online but that doesn't seem to be the case. What's also strange is that running a comet locally lets me successfully `|hi` the parent star of `~fodbyr-nosmyr`, I am even able to join `~dopzod/urbit-help`, but the messages I tried sending wouldn't go through, even from the comet.

I am starting to think that my pier and maybe my urbit installation are in some weird broken state. What would be the right way to reset it all and start again?

**Comment by Fang-** [comment]
> I've tried resetting my public keys multiple times too but that didn't help.

This might be part of the problem. If you do this on-chain using Bridge, you also need to update the keys in your running ship. You do this by running `|rekey '0wwhatever'` from dojo, replacing that string with the keyfile contents/string that Bridge gives you when you "generate keyfile". (Though, `|rekey` may give you a hard time about not exactly incrementing by one revision. You may want to comment out [there lines](https://github.com/urbit/arvo/blob/master/lib/hood/helm.hoon#L84-L86).)

Then, it's a matter of ships you want to talk with being up-to-date enough with chain state to know about the keys you're using, and you being up to date enough to know theirs. You have no control over others, but at least you can check your own ship! Run `-:.^(* %j /=eth-status=)`, and make sure the number matches the latest block number [here](https://etherscan.io/address/azimuth.eth#events) (currently `8.039.973`). If it doesn't, you can `|start %eth-manage` and then `:eth-manage %look-ethnode` to get up to speed.

There's more drastic measures we can try if that doesn't work, but give that a shot first. I did manage to `|hi` your sponsor (~panten) successfully, so at least that's running!

**Comment by alexeyzab** [comment]
Thank you for the suggestions!

Interestingly enough, when I got to my desktop to try this out I noticed that I can successfully `|hi` `zod`, `nem`, `panten` and `dopzod`. But attempting to `;join ~dopzod/urbit-help` does nothing still, it prints out this:

```
--------------| ;join ~dopzod/urbit-help
```

I tried using rekey which resulted in it running successfully, but I haven't noticed any difference after running it.

Tried checking if my ship is up to date, the number does indeed match the latest block number.
